core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -204,12 +204,14 @@ public URIStatus getStatus(AlluxioURI path, GetStatusOptions options)
   @Override
   public void loadMetadata(AlluxioURI path)
       throws FileDoesNotExistException, IOException, AlluxioException {
+    LOG.debug("loadMetadata {} by default LoadMetadataOptions", path);
     loadMetadata(path, LoadMetadataOptions.defaults());
   }
 
   @Override
   public void loadMetadata(AlluxioURI path, LoadMetadataOptions options)
       throws FileDoesNotExistException, IOException, AlluxioException {
+    LOG.debug("loadMetadata {} by LoadMetadataOptions {}", path, options);
```

Comment:
I think this can be "loaded metadata {} with options {}"

Also, `createDirectory()` uses LOG.info(). Maybe we should stay consistent?